api/src/main/kotlin/edu/wgu/osmt/collection/CollectionRepository.kt

```diff
@@ -197,6 +206,30 @@ class CollectionRepositoryImpl @Autowired constructor(
         return daoObject
     }
 
+    override fun remove(uuid: String): ApiBatchResult {
+
+        val collectionFound = findByUUID(uuid)
+
+        if (collectionFound != null) {
+            collectionSkillsTable.deleteWhere { collectionSkillsTable.collectionId eq collectionFound.id }
+            collectionFound?.let { collectionEsRepo.delete(it.toDoc()) }
```

Comment:
Should these 2 deletes (db repo and ES repo) be in a transaction? What does it cost if yes? What's the risk if no?

Reply:
changed logic here